cmd/luna/app/import.go

```diff
@@ -9,8 +9,31 @@ import (
 	"github.com/quay/claircore/libvuln"
 	"github.com/quay/zlog"
 	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
 )
 
+func NewImportCommand() *cobra.Command {
+	var importFileName string
```

Comment:
📝 **[sonarqube]** <[go:S1135](http://localhost:19000/project/issues?id=luna-code-enhancement&issues=AXgQ3tLRvT_5avwPlAaj)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Complete the task associated to this TODO comment.